initializers/device_types.yml

```diff
@@ -1,21 +1,21 @@
-# - model: Model 1
-#   manufacturer: Manufacturer 1
-#   slug: model-1
-#   u_height: 2
-#   custom_field_data:
-#     text_field: Description
-# - model: Model 2
-#   manufacturer: Manufacturer 1
-#   slug: model-2
-#   custom_field_data:
-#     text_field: Description
-# - model: Model 3
-#   manufacturer: Manufacturer 1
-#   slug: model-3
-#   is_full_depth: false
-#   u_height: 0
-#   custom_field_data:
-#     text_field: Description
+# # - model: Model 1
```

Comment:
Why did you comment this section? (I mean the extra comment level of course)

Reply:
Sorry for that. It's a mistake done when I switched comments in what I've added to '# #' instead of '##'. Will fix this now